TeamCode/src/main/java/org/firstinspires/ftc/teamcode/DriveTrain.java

```diff
@@ -0,0 +1,646 @@
+        power = Math.sqrt((x*x) + (y*y));
+        switch(zone) {
+            case 0:
+                setMotorPower(power, 0, z);
+                telemetry.addData("Current Movement", Direction.STRAFERIGHT);
+                break;
+            case 1:
+                setMotorPower(power, (power*.41), z);
+                telemetry.addData("Current Movement", Direction.DSTRAFERIGHTUP);
+                break;
+            case 2:
+                setMotorPower(power, power, z);
+                telemetry.addData("Current Movement", Direction.DRIGHTUP);
+                break;
+            case 3:
+                setMotorPower((power*.41), power, z);
+                telemetry.addData("Current Movement", Direction.DUPSTRAFERIGHT);
+                break;
+            case 4:
+                setMotorPower(0, power, z);
+                telemetry.addData("Current Movement", Direction.FORWARD);
+                break;
+            case 5:
+                setMotorPower(-(power*.41), power, z);
+                telemetry.addData("Current Movement", Direction.DUPSTRAFELEFT);
+                break;
+            case 6:
+                setMotorPower(-power, power, z);
+                telemetry.addData("Current Movement", Direction.DLEFTUP);
+                break;
+            case 7:
+                setMotorPower(-power, (power*.41), z);
+                telemetry.addData("Current Movement", Direction.DSTRAFELEFTUP);
+                break;
+            case 8:
+                setMotorPower(-power, 0, z);
+                telemetry.addData("Current Movement", Direction.STRAFELEFT);
+                break;
+            case 9:
+                setMotorPower(-power, -(power*.41), z);
+                telemetry.addData("Current Movement", Direction.DSTRAFELEFTDOWN);
+                break;
+            case 10:
+                setMotorPower(-power, -power, z);
+                telemetry.addData("Current Movement", Direction.DLEFTDOWN);
+                break;
+            case 11:
+                setMotorPower(-(power*.41), -power, z);
+                telemetry.addData("Current Movement", Direction.DDOWNSTRAFELEFT);
+                break;
+            case 12:
+                setMotorPower(0, -power, z);
+                telemetry.addData("Current Movement", Direction.BACKWARD);
+                break;
+            case 13:
+                setMotorPower((power*.41), -power, z);
+                telemetry.addData("Current Movement", Direction.DDOWNSTRAFERIGHT);
+                break;
+            case 14:
+                setMotorPower(power, -power, z);
+                telemetry.addData("Current Movement", Direction.DRIGHTDOWN);
+                break;
+            case 15:
+                setMotorPower(power, -(power*.41), z);
+                telemetry.addData("Current Movement", Direction.DSTRAFERIGHTDOWN);
+                break;
+        }
+    }
+
+    public void Drive(Direction direction, double power){
```

Comment:
#fix spacing and capitalization `public void Drive(Direction direction, double power){` -> `public void drive(Direction direction, double power) {`